internal/cmd/modules/modules.go

```diff
@@ -135,6 +151,43 @@ func listManagedModules(cfg *modulesConfig) ([]string, clierror.Error) {
 	return moduleNames, nil
 }
 
+func listInstalledModules(cfg *modulesConfig) ([]string, clierror.Error) {
+	resp, err := http.Get("https://raw.githubusercontent.com/kyma-project/community-modules/main/model.json")
+	if err != nil {
+		return nil, clierror.Wrap(err, clierror.New("while getting modules list from github"))
+	}
+	defer resp.Body.Close()
+
+	var template model.Module
+
+	template, respErr := handleResponse(err, resp, template)
+	if respErr != nil {
+		return nil, clierror.WrapE(respErr, clierror.New("while handling response"))
+	}
+
+	var out []string
+	for _, rec := range template {
+		short := strings.Split(rec.Versions[0].ManagerPath, "/")
+		version := rec.Versions[0].Version
+		deployment, err := cfg.KubeClient.Static().AppsV1().Deployments("kyma-system").Get(cfg.Ctx, short[len(short)-1], metav1.GetOptions{})
+		if err != nil && !errors.IsNotFound(err) {
+			msg := "while getting the " + short[len(short)-1] + " deployment"
+			return nil, clierror.Wrap(err, clierror.New(msg))
+		}
+		if !errors.IsNotFound(err) {
+			depVersion := strings.Split(deployment.Spec.Template.Spec.Containers[0].Image, "/")
```

Comment:
IMO `depVersion` contains image name and tag, not deployment version. Could you find better name?

Reply:
Changed to `deploymentImage`